src/app/src/main/kotlin/org/vaccineimpact/orderlyweb/app_start/RouteConfig.kt

```diff
@@ -19,9 +19,8 @@ interface RouteConfig
 object APIRouteConfig : RouteConfig
 {
     override val endpoints: List<EndpointDefinition> =
-            ReportRouteConfig.endpoints
+            GitRouteConfig.endpoints.plus(ReportRouteConfig.endpoints)
```

Comment:
While looking into this I also found an ambiguity in the API:

`/api/v1/reports/git/status` was matching `/api/v1/reports/:key/status`

Rather than changing the API spec I've just changed the order the routes are specified in so the `git` one gets matched first.